lib/routes/njupco/handler-func.ts

```diff
@@ -0,0 +1,53 @@
+import * as cheerio from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+export async function handler() {
+    interface NewsItem {
+        title: string,
+        link: string,
+        pudDate: Date | null,
+        description: string,
+    }
+    const TARGET_URL = 'http://www.njupco.com/news/press';
+    const $ = await cheerio.fromURL(TARGET_URL);
```

Comment:
[Use `ofetch` properly](https://docs.rsshub.app/joinus/new-rss/start-code) instead of cheerio.fromURL()